src/watcher.ts

```diff
@@ -1,39 +1,61 @@
 import chokidar from 'chokidar';
 import { exec, ChildProcess } from 'child_process';
+import type Serverless from 'serverless';
 
-export type Config = {
+type ConfigItem = {
   path: string | string[];
-  command: string;
-}[];
+  command?: string;
```

Comment:
Optional: If you're in the area, would you consider changing this to `command?: string[]`, so the API interface is more consistent between commands and hooks. Looks like you've already done great work refactoring out runCommand to a function that probably can just be called multiple times like we are for hooks.

Reply:
Yeah I can do that. I personally don't even have a use case for having multiple hooks invoked, but my reasoning here was that, even though the plugin supported only one command, it would be pretty easy to make that invoke multiple commands in userland. I mean all you have to do is invoke another script that then launches multiple commands. For hooks it's more cumbersome to do userland,  as you'd need to create a whole serverless plugin that invokes those multiple hooks from the hook invoked by this plugin.

That said, array support for `command` is an easy thing to add here so why not.

One thing though, shoud the commands be executed sequentially, as in
```javascript
for (const command of commands) {
  await runCommand(command)
}
```
or in parallel?
```javascript
await Promise.all(commands.map(runCommand))
```

You may notice that I made the hooks spawn only after commands have closed. This was very much intentional, because I'm envisioning that I'd first run a build command with the `command` option, and only after that's finished do I want to notify serverless-offline that it should invalidate cache. But what about the commands?

I'd lean to running them sequentially, by the "principle of least surprise", it feels like a footgun if
```yaml
  command:
    - thing a
    - thing b
```
thing b would _sometimes_ complete first and sometimes a. If users want to run them parallel for some perf benefit, they can always extract that to a script and do whatever bash schenanigans in there.

What do you think?

Reply:
Related, with an implementation that I thought would be good and simple here,

```javascript
          for (const command of normalizeStringArray(c.command)) {
            await runCommand(command, eventType, eventPath);
          }
          for (const hook of normalizeStringArray(c.hook)) {
            runHook(hook);
          }
```
I'm running into couple of ESLint errors.
```
iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.eslint[no-restricted-syntax](https://eslint.org/docs/rules/no-restricted-syntax)

Unexpected `await` inside a loop.eslint[no-await-in-loop](https://eslint.org/docs/rules/no-await-in-loop)
```

Is it ok if I disable these rules for the project, or at least for the function? I can refactor the loop to `.reduce` that waits previous promise before running the next, but the benefit of doing so seems questionable to me. IMHO the loop version is much easier to read. The note about regenerator-runtime also doesn't apply here because the current build doesn't downlevel async/await and doesn't include it anyway. There's also existing async/await in the project so having that here wouldn't raise the required Node version limit from whatever it is now.

Reply:
I think running them sequentially makes the most sense, and would be what I'd expect as a user.

Agreed the for-loop is the easier way to go here. Happy for you to disable the eslint rule for this line :)

Reply:
Array support for `command` and `hooks` -> `hook` rename were implemented in 3e8c08b3368dca